controllers/disruption_rollout_controller.go

```diff
@@ -32,6 +35,18 @@ func (r *DisruptionRolloutReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	instance := &chaosv1beta1.DisruptionRollout{}
 	randSource := rand.New(rand.NewSource(time.Now().UnixNano()))
 
+	// reconcile metrics
+	r.handleMetricSinkError(r.MetricsSink.MetricReconcile([]string{"rolloutName:" + instance.Name, "namespace:", instance.Namespace}))
```

Comment:
I don't think those are the right tags to send here? Maybe I'm missing something / not understanding something?
Or maybe it's the other way and this is right and the ones on the cron controller and chaos controller are wrong? 😅

Reply:
No youre right, these are incorrect tags. Should be the name of the App, not the individual rollouts that go through the Reconcile loop.